tests/wpt/metadata/MANIFEST.json

```diff
@@ -29821,6 +29821,24 @@
     "deleted": [],
     "items": {
       "testharness": {
+        "dom/nodes/Node-contains.xml": [
```

Comment:
I think the changes in this file are unrelated. It's probably best to revert this particular file and do this in another patch.